setup/README.md

```diff
@@ -1,21 +1,28 @@
 # Use the Command Line to Manage ILIAS
 
-The ILIAS command line app can be called via `php setup\setup.php`. It contains three
-commands to manage ILIAS installations:
+The ILIAS command line app can be called via `php setup\setup.php`. It contains four
+main commands to manage ILIAS installations:
 
-* `install` will [set an installation up](#install-ilias)
-* `update` will [update an installation](#update-ilias)
+* `install` will [set an installation up including plugins](#install-ilias)
+* `update` will [update an installation including plugins](#update-ilias)
 * `build-artifacts` [recreates static assets](#build-ilias-artifacts) of an installation
 * `reload-control-structure` [rebuilds structure information](#build-ilias-artifacts) of an installation
 
+`install` and `update` also supply switches and options to granular control what to install or update.
+
+* `install --no_plugins` will [set an installation up without plugins](#install-ilias-without-plugins)
```

Comment:
we decided on `--no-plugin` with a "-" during JF, there are several places here which have a "_" in the subcommand.

Reply:
Ty Fabian, i fixed it.